modules/migrations/github.go

```diff
@@ -214,6 +214,11 @@ func (g *GithubDownloaderV3) convertGithubRelease(rel *github.RepositoryRelease)
 		name = *rel.Name
 	}
 
+	var email string
+	if rel.Author.Email != nil {
+		email = *rel.Author.Email
```

Comment:
Is it possible that this e-mail comes in the format `<username>@users.noreply.github.com`? Would that be a problem?

Reply:
@guillep2k If the author public his email, this email will be not nil, so we could copy it. If he hides his email on github, we will receive nil here. So I don't think this is a problem.